src/main/python/api/house_service.py

```diff
@@ -0,0 +1,18 @@
+import flask
+from flask import jsonify
+import pymysql
+import sys
+from pathlib import Path
+
+sys.path.insert(1, str(Path(__file__).parent.parent))
+from datastore import house_dao
+
+app = flask.Flask(__name__)
+app.config["DEBUG"] = True
+
+
+@app.route('/', methods=['GET'])
+def fetch_all_houses():
+    return jsonify(house_dao.get_all_houses())
```

Comment:
With Flask (but also any HTTP framework), you always want to return the response *and* the status code in a tuple, e.g `return jsonify(house_dao.get_all_houses()), 200`.

Reply:
Sure - fixed.